drivers/sensor/apds9960/apds9960.h

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SENSOR_APDS9960_H_
+#define _SENSOR_APDS9960_H_
+
+#include <device.h>
```

Comment:
Do you really need all these includes here? I think it would be cleaner to keep just what is needed and move the rest to the .c file.